src/utils/validators/index.ts

```diff
@@ -12,4 +12,24 @@ const validateRegexp = (regexp: string): boolean => {
   }
 };
 
-export { validateRegexp };
+/**
+ * `getRegexpError` returns the error message for a given regexp
+ * @param regexp - the regexp to test
+ * @returns
+ */
+const getRegexpError = (regexp: string): string => {
+  try {
+    // eslint-disable-next-line no-new
+    new RegExp(regexp);
+    return "";
+  } catch (e: any) {
+    console.log(e.message);
+    const message = e.message.replace(
```

Comment:
[nit] Could be helpful to mention we're removing this string from the beginning of the error message